core/src/main/web/outputdisplay/bko-plot/bko-plot.js

```diff
@@ -841,7 +841,13 @@
             if (line.legend == null || line.legend === "") { continue; }
             var unit = $(legendLineUnit).appendTo(legend)
               .attr("id", "legend_" + id)
-              .addClass("plot-legendline");
+              .addClass("plot-legendline")
+              .mouseenter(function(e){
```

Comment:
I think we need to make sure we clean up these event listeners, otherwise we are going to leak